packages/react/src/dialog-footer/DialogFooter.css.ts

```diff
@@ -0,0 +1,17 @@
+import { theme } from "../styles";
+import { recipe, style } from "../vanilla-extract";
+
+export const footer = recipe({
+  base: [
+    {
+      flexDirection: "row",
+      gap: "md",
+      justifyContent: "end",
+      px: "24",
+      py: "20",
+    },
+    style({
+      borderTop: `1px solid ${theme.colors["border.secondary"]}`,
+    }),
```

Comment:
i've added sprinkles for individual borders #154

so we can replace this with 
```suggestion
      borderTop: "1",
    },
```